scripts/delivery.json

```diff
@@ -48,9 +48,26 @@
         { "name": "mobile_number", "type": "string", "required": false },
         { "name": "email_id", "type": "string", "required": false }
       ]
+    },
+    
+    "delivery_form" : {
+    	"fields" : [
+        { "name": "order_number", "type": "string" },
+        { "name": "merchant_id", "type": "string" },
+        { "name": "origin", "type": "address" },
+        { "name": "destination", "type": "address" },
+        { "name": "contact_info", "type": "contact" }	
+    	]
+    },
+    
+    "error": {
+      "fields": [
+        { "name": "code", "type": "[string]" },
```

Comment:
the code would be a string and the messages could be multiple. So only message will be sequence

Reply:
I have changed the same.